### PR TITLE
Implements errors visual display bot creation / signin

### PR DESCRIPTION
**Pull request by JulienUsson**
[Issue 53](https://github.com/Opla/front/issues/53)

### CODE REVIEW

File: src/shared/containers/createAssistant.jsx [hunk 1]
```diff
@@ -215,23 +218,27 @@ class CreateAssistant extends Component {
...
                 }}
+                required
               />
             </div>
             <div>
               <Select
                 label="Choose language"
                 onSelected={this.handleLanguageChange}
                 style={{ width: "400px" }}
+                required
```
Comment (mikbry) [inline src/shared/containers/createAssistant.jsx]: Not sure if required is implemented in ZRMC Select

Comment (JulienUsson) [reply src/shared/containers/createAssistant.jsx]: It's not, i was going to ask the question.
It's not defined in the https://material.io/components/web/ doc
Idk if we want to implement it or use another strategy

Comment (willdurand) [reply src/shared/containers/createAssistant.jsx]: FWIW, `required` is a HTML5 attribute. There is no reason not to support it in zrmc.

Comment (mikbry) [reply src/shared/containers/createAssistant.jsx]: "required" should be implemented in ZRMC. Even if they don't implement it in Material Components.

**Comment by mikbry** [comment]
Test cases must be implemented, see dashboard.test.js as a example.

**Pull request merged**